src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunctionDecorator.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.Operations.Indexing;
+
+namespace Microsoft.Health.Dicom.Functions.Indexing
+{
+    public sealed class ReindexDurableFunctionDecorator
+    {
+        private readonly ReindexDurableFunction _function;
+
+        public ReindexDurableFunctionDecorator(ReindexDurableFunction function)
+            => _function = EnsureArg.IsNotNull(function, nameof(function));
+
+        [FunctionName(nameof(StartReindexingInstancesAsync))]
```

Comment:
As you can see, I am creating the HTTP functions in this project by using the various attributes to specify auth, path, etc